### PR TITLE
Resolved the documentation errors

### PR DESCRIPTION
**Pull request by ArshadAriff**

### CODE REVIEW

**Comment by ArshadAriff** [comment]
I have resolved the errors with the documentation

File: docs/docs/scheme-intro/input-output.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ If you first evaluated the code that writes into a file `test-2.scm` you should
 was written into it. Note that `read` was called two times and read two expressions (two strings).
 
 Same as with output-port you need to call `close-input-port` procedure to close the input port. But
-Scheme, same as with output port have handy function that allow to close the port automagically.
+Scheme, same as with output port have handy function that allow to close the port automatically.
```
Comment (jcubic) [inline docs/docs/scheme-intro/input-output.md:162]: You didn't fixed this:

This is not an error, see ["automagically" in Jargon File](http://www.catb.org/jargon/html/A/automagically.html).

**Review by jcubic — COMMENTED** [review]
Please don't close the PR. All commits you made will be added here. PR is dynamic, it's not frozen when you create it.

**Comment by ArshadAriff** [comment]
Sir i changed all the typos are these enough sir! anything more?

**Comment by jcubic** [comment]
I'm not gonna marge this, you did change random stuff like syntax of the code, replacing single quote with double quote or replacing `*` with `-`. The task was to change grammar of the text, to change the syntax of Markdown and code.

Also, I still, see that you changed `automagically` to `automatically`, that I specifically asked not to do.

**Comment by ArshadAriff** [comment]
Sir i really want to contribute please guide me on exactly what to do!

**Comment by jcubic** [comment]
Remove the changes to the Markdown syntax and source code. And remove the change to replace "automagically" with "automatically".

**Pull request closed without merging**